cspell.json

```diff
@@ -1439,6 +1451,7 @@
     "SEGDPL",
     "SEGLIM",
     "selinuxenabled",
+    "seperate",
```

Comment:
That's an actual mispelled word

Reply:
Not sure i did that one but I will fix it

Reply:
fixed